tidb-cloud/tidb-cloud-console-auditing.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Web Console Audit Logging
+Summary: Learn about how to audit operations on the TiDB Cloud Web Console.
+---
+
+# Web Console Audit Logging
+
+TiDB Cloud provides you with the audit log function for Web Console operations, which can record the history of user access details (such as user login to Web Console, cluster creation, etc.).
+
+> **Note:**
+>
+> Currently, the **audit logging** feature is experimental. The interface and output are subject to change.
+
+To assess the effectiveness of user access policies and other information security measures of your organization, it is a security best practice to conduct a periodic analysis of the DbaaS audit logs.
+
+## Specify auditing filter rules
+
+The web console audit logging feature is enabled by default, and all user operations on the Web Console will be recorded in the audit logs.
+Audit logs cover the following event types:
+
+| Audit Event Type | Description |
+|---|---|
+| AuditEventSignIn | sign in |
```

Comment:
This table has many duplicated Audit Event Types with the table in the "Audit log fields" section. Shall we only retain one table?

Reply:
1. The meanings of these two tables are different. I have communicated with caiqian before and the conclusion is that both tables are reserved.
2. The first table describes the audit coverage scenarios, and configurable filter conditions may be added later; the second table describes the record fields in each scenario.